sku.go

```diff
@@ -368,6 +368,17 @@ func (s *SKU) GetName() string {
 	return *s.Name
 }
 
+// GetFamilyName returns the family name of this resource sku. It normalizes pointers
+// to the empty string for comparison purposes. For example,
+// "standardDSv2Family" for a virtual machine.
+func (s *SKU) GetFamilyName() string {
```

Comment:
could you add a unit test on this function in data_test.go? https://github.com/Azure/skewer/blob/main/data_test.go